docs/getting-started/quickstart.rst

```diff
@@ -78,9 +78,15 @@ a target:
 
 .. ipython:: python
 
-    my70mPass = au.Pass(36, "10_zone", 122, 70.0)
+    my70mPass = au.Pass(36, my720target)
     print(my70mPass.max_score())
 
+We can also directly construct a Pass and Target at the same time using the `at_target` constructor
```

Comment:
I know it's pedantic, but I'd prefer a slight rephrase here as we technically don't construct a `Target` as a separate entity, rather we bypass the need to create an intermediate `Target` to provide to `Pass` by providing the target data directly to create a `Pass`.